packages/shared/src/lib/featureManagement.ts

```diff
@@ -46,6 +46,7 @@ const feature = {
   tagsOnCard: new Feature('tags_on_card', false),
   onboardingMostVisited: new Feature('onboarding_most_visited', false),
   shareExperience: new Feature('share_experience', false),
+  sidebarToggle: new Feature('sidebar_toggle', true),
```

Comment:
I think a better name would be `sidebarOpen`, since that's what it actually does no? 🤔

Reply:
I agree with Milan here on the naming toggle not being ideal - I would actually suggest `sidebarClosed`, this way we can keep default as we always do to false and have it true for experiment.

I can change the growthbook experiment if we agree on this

Reply:
I just followed the convention to do camelCasing on the actual value. If we want we should probably change its feature name too.

Reply:
Yeah I was referring to both, I'll change the growthbook feature to sidebar_closed, let's change the naming here as well for both 🙌